doc/apiary/iotagent.apib

```diff
@@ -93,9 +93,9 @@ Mandatory fields are identified in every operation.
 
 
 
-### Retrieve a service [GET]
+### Retrieve a configuration group [GET]
 
-With Fiware-ServicePath you can retrieve a subservice or all subservices.
+Retrieve a group configuration.
```

Comment:
group configuration -> configuration group

Reply:
fixed in 8ba00f67db1897dddfea5c1bc64bdd0d9ac18aeb